akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEventParser.scala

```diff
@@ -22,6 +22,58 @@ import scala.util.control.NonFatal
 
 private object ServerSentEventParser {
 
+  private class Builder {
```

Comment:
If you make the builder `final` you give the JVM optimizer more info to inline the calls to the methods.